sklearn/cluster/tests/test_optics.py

```diff
@@ -16,6 +16,7 @@
 from sklearn.metrics.pairwise import pairwise_distances
 from sklearn.utils import shuffle
 from sklearn.utils._testing import assert_allclose, assert_array_equal
```

Comment:
I think the changes in this file might not be required in this PR. There is a separate PR open to treat `test_optics.py` - #27104